### PR TITLE
Separate mail template for incomplete payment notifications

### PR DESCRIPTION
**Pull request by julianrother**
Closes https://github.com/pretix/pretix/issues/2819

### CODE REVIEW

File: src/pretix/base/settings.py [hunk 1]
```diff
@@ -2002,6 +2002,24 @@ def unserialize(cls, s):
 You can view the payment information and the status of your order at
 {url}
 
+Best regards,
+Your {event} team"""))
+    },
+    'mail_subject_order_incomplete_payment': {
+        'type': LazyI18nString,
+        'default': LazyI18nString.from_gettext(gettext_noop("Your order received an incomplete payment: {code}")),
```
Comment (raphaelm) [inline src/pretix/base/settings.py:2010]: I'll suggest a shorter version:

```suggestion
        'default': LazyI18nString.from_gettext(gettext_noop("Incomplete payment received: {code}")),
```

File: src/pretix/base/settings.py [hunk 2]
```diff
@@ -2002,6 +2002,24 @@ def unserialize(cls, s):
...
+    },
+    'mail_text_order_incomplete_payment': {
+        'type': LazyI18nString,
+        'default': LazyI18nString.from_gettext(gettext_noop("""Hello,
+
+we did not yet receive a full payment for your order for {event}.
```
Comment (raphaelm) [inline src/pretix/base/settings.py:2016]:
```suggestion
we received a payment  for your order for {event}.
Unfortunately, the received amount is smaller than the full amount required. Your order is therefore still considered unpaid.
```

**Review by raphaelm — COMMENTED** [review]
Draft looks good on first glance.

Do you think it would be worth the effort adding a placeholder for the amount that is still missing / the amount that has been received into the text?

I currently don't get why the test failure happens, could you rebase against master if it still happens? It's in a test that is related to payment reminders, so I'm not 100% sure it's unrelated.

**Comment by raphaelm** [comment]
I felt bad for merging a separate PR today that created merge conflicts with this one, after this one already was around for a while, so I took the liberty of cleaning it up and finishing this one. Maybe @pc-coholic or @wiffbi can have another short look, but I think this one is then ready to go.

**Review by wiffbi — APPROVED** [review]
Looks good, just some minor comments regarding text though I am not a native speaker, so take with a grain of salt.

**Comment by raphaelm** [comment]
Thanks a lot everyone!

**Comment by julianrother** [comment]
Thank you, too!

**Pull request merged**